moveos/moveos-store/src/lib.rs

```diff
@@ -291,3 +308,13 @@ impl StateResolver for MoveOSStore {
         self.statedb.resolve_list_state(handle, cursor, limit)
     }
 }
+
+impl AccumulatorStore for MoveOSStore {
+    fn get_accumulator(&self) -> Result<Option<Accumulator>> {
+        self.get_accumulator_store().get_accumulator()
+    }
+
+    fn save_accumulator(&self, accumulator: Accumulator) -> Result<()> {
+        self.get_accumulator_store().save_accumulator(accumulator)
```

Comment:
We can not directly save the whole accumulator. 

### Refs 

* https://github.com/starcoinorg/starcoin/tree/master/commons/accumulator
* https://github.com/aptos-labs/aptos-core/tree/main/storage/accumulator/src

Reply:
Thank you for the resources. They'd help

Reply:
Hi, @jolestar I have added new commits.

Most part of the code reuse starcoin implementation, https://github.com/starcoinorg/starcoin/tree/master/commons/accumulator. I modified the code to fit into the moveos codebase and successfully run.

Do you think it's okay and I can proceed with changing the PR status from draft?